arduino-ide-extension/src/node/i18n/arduino-localization-contribution.ts

```diff
@@ -3,150 +3,45 @@ import {
   LocalizationRegistry,
 } from '@theia/core/lib/node/i18n/localization-contribution';
 import { injectable } from '@theia/core/shared/inversify';
+import { join } from 'path';
 
 @injectable()
 export class ArduinoLocalizationContribution
   implements LocalizationContribution
 {
-  async registerLocalizations(registry: LocalizationRegistry): Promise<void> {
-    registry.registerLocalizationFromRequire(
-      'af',
-      require('../../../build/i18n/af.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'en',
-      require('../../../build/i18n/en.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'fr',
-      require('../../../build/i18n/fr.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'ko',
-      require('../../../build/i18n/ko.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'pt-br',
-      require('../../../build/i18n/pt.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'uk_UA',
-      require('../../../build/i18n/uk_UA.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'ar',
-      require('../../../build/i18n/ar.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'es',
-      require('../../../build/i18n/es.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'he',
-      require('../../../build/i18n/he.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'my_MM',
-      require('../../../build/i18n/my_MM.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'ro',
-      require('../../../build/i18n/ro.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'zh-cn',
-      require('../../../build/i18n/zh.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'bg',
-      require('../../../build/i18n/bg.json')
-    );
-
-    registry.registerLocalizationFromRequire(
-      'eu',
-      require('../../../build/i18n/eu.json')
-    );
+  // 0. index: locale
+  // 1. index: optional JSON file to `require` (if differs from the locale)
+  // If you touch the locales, please keep the alphabetical order. Also in the `package.json` for the VS Code language packs. Thank you! ❤️
+  // Note that IDE2 has more translations than available VS Code language packs. (https://github.com/arduino/arduino-ide/issues/1447)
+  private readonly locales: ReadonlyArray<[string, string?]> = [
+    ['bg'],
+    ['cs'],
+    ['de'],
+    ['es'],
+    ['fr'],
+    ['hu'],
+    // ['id'], Does not have Transifex translations, but has a VS Code language pack available on Open VSX.
+    ['it'],
+    ['ja'],
+    ['ko'],
+    ['nl'],
+    ['pt-br', 'pt'],
+    ['pl', 'pl'],
```

Comment:
```suggestion
    ['pl'],
    ['pt-br', 'pt'],
```

- Use alphabetical order as specified in the comment
- Remove superfluous second element for Polish